### PR TITLE
data: Palakkgoyal

### PR DESCRIPTION
**Pull request by Palakkgoyal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5814"></a>

### CODE REVIEW

File: data/Palakkgoyal.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/Palaktwts",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/Palakkgoyal",
+      "icon": "FaGithub"
+    },
```
Comment (khushalarora) [inline data/Palakkgoyal.json:17]: Please remove the extra **' Comma '** from this line :+1: 
```Suggestion
    }
```

File: data/Palakkgoyal.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Palak Goyal",
+  "displayStatsPublic": true,
```
Comment (khushalarora) [inline data/Palakkgoyal.json:3]: You can delete this line, it is not required anymore :+1: 
```Suggestion
```

File: data/Palakkgoyal.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  "type": "personal",
+  "bio": "Palak is a frontend developer and besides programming she loves cooking, meditating and travelling",
+  "avatar": "https://github.com/Palakkgoyal.png",
```
Comment (khushalarora) [inline data/Palakkgoyal.json:6]: you can delete this line, it is not required anymore :+1: 
```Suggestion
```

**Review by khushalarora — CHANGES_REQUESTED** [review]

**Review by khushalarora — CHANGES_REQUESTED** [review]
Hi @Palakkgoyal, thank you for contributing :tada:

**Review by accodes21 — APPROVED** [review]

**Review by SandeepKrSuman — APPROVED** [review]
Thanks, @Palakkgoyal 🎉 Looks good to me!

**Review by khushalarora — APPROVED** [review]
Thank you for contributing  :tada:

**Review by SurajPratap10 — APPROVED** [review]
Thanks for contributing 👍

**Review by SaraJaoude — APPROVED** [review]
looks good

**Pull request merged**